Mathlib/Logic/Function/Basic.lean

```diff
@@ -69,6 +69,10 @@ theorem ne_iff {β : α → Sort _} {f₁ f₂ : ∀ a, β a} : f₁ ≠ f₂ 
 protected theorem Bijective.injective {f : α → β} (hf : Bijective f) : Injective f := hf.1
 protected theorem Bijective.surjective {f : α → β} (hf : Bijective f) : Surjective f := hf.2
 
+-- Porting note : mathlib3port wrote `().Injective` instead of `().injective`.
+#align function.bijective.injective Function.Bijective.injective
+#align function.bijective.surjective Function.Bijective.surjective
```

Comment:
I'd be surprised if these #aligns actually fix that issue. It would be nice to test in oneshot.

Reply:
Currently oneshot does not work for me and I don't have the time rn to make it work. If it doesn't fix it, it porbably doesn't hurt anybody either.

Reply:
Others have added these aligns in other PRs by now, so I guess it's moot.